src/components/TextLink.js

```diff
@@ -31,6 +31,7 @@ const defaultProps = {
 };
 
 const TextLink = (props) => {
+    const rest = _.omit(props, _.keys(propTypes));
```

Comment:
I just want to make sure I understand what is happening here...

When we call `TextLink` and pass props, the `rest` variable is going to contain all the props that are not in the `propTyles` defined above.

Then, we are adding the props that are in `rest` to the props that we are sending to `Text` below.

Is that all correct?